src/application/deployer/index.go

```diff
@@ -97,19 +105,17 @@ func validateConfigs(deployConfig deploy.Config) error {
 	return nil
 }
 
-func deployApplication(deployConfig deploy.Config, imagesMap map[string]string) error {
+func deployApplication(deployConfig deploy.Config, imagesMap map[string]string, secrets types.Secrets) error {
 	fmt.Fprintln(deployConfig.Writer, "Retrieving remote state...")
 	err := terraform.RunInit(deployConfig)
 	if err != nil {
 		return err
 	}
 
-	fmt.Fprintln(deployConfig.Writer, "Retrieving secrets...")
-	secrets, err := aws.ReadSecrets(deployConfig.AwsConfig)
-	if err != nil {
-		return err
-	}
-
 	fmt.Fprintln(deployConfig.Writer, "Applying changes...")
-	return terraform.RunApply(deployConfig, secrets, imagesMap, deployConfig.DeployServicesOnly)
+	err = terraform.RunApply(deployConfig, secrets, imagesMap, deployConfig.DeployServicesOnly)
+	if err != nil && strings.Contains(err.Error(), "exit status") {
+		return nil
+	}
```

Comment:
Thoughts on checking on the err containing the "exit status" in the `terraform.RunApply`. Since its specific to that I think it belongs there and we can comment that it is expected to exit with non-zero status when the user choose to abandon the deployment